spec/core/genesis.md

```diff
@@ -0,0 +1,32 @@
+# Genesis
+
+The genesis file is meant to be the starting point of chain. Tendermint's genesis is minimal because it is not aware of application state. An application will populate the genesis file with their own required fields.
```

Comment:
```suggestion
The genesis file is the starting point of a chain. Tendermint's genesis is minimal because it is not aware of application state. An application will populate the genesis file with their own required fields.
```

Reply:
yes, I will add it in. When generating genesis via Tendermint-go the appState field is not generated.

Reply:
You make it sound like there is a distinction between app genesis and tendermint genesis. I thought they were one and the same file